ui/src/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestDetails/BorrowingRequestPayment.js

```diff
@@ -74,13 +74,15 @@ class Payment extends React.Component {
 export class BorrowingRequestPayment extends React.Component {
   render() {
     const brwReq = this.props.brwReq;
+    const paymentInfo = !_isEmpty(brwReq.payment);
     return (
       <>
         <Header as="h3" attached="top">
           Payment information
         </Header>
         <Segment attached className="bo-metadata-segment" id="payment-info">
-          {!_isEmpty(brwReq.payment) && <Payment brwReq={brwReq} />}
+          {paymentInfo && <Payment brwReq={brwReq} />}
+          {!paymentInfo && `No payment information`}
```

Comment:
how about `<InfoMessage>` here?

Reply:
done, but I went for a normal message (no blue), it looked better IMHO.